apps/web/models.py

```diff
@@ -17,3 +17,33 @@ def __unicode__(self):
     class Meta:
         app_label = 'web'
         db_table = 'contact'
+
+
+class Team(models.Model):
+    """Model representing details of Team"""
+
+    # Team Type Options
+    CORE_TEAM = 'Core Team'
+    CONTRIBUTOR = 'Contributor'
+
+    TEAM_TYPE_OPTIONS = (
+        (CORE_TEAM, CORE_TEAM),
+        (CONTRIBUTOR, CONTRIBUTOR),
+    )
+
+    name = models.CharField(max_length=100,)
```

Comment:
can you please remove the trailing comma after 100. Infact please remove it for every other field (email, github_url,...)